compute_sdk/PyPI.md

```diff
@@ -0,0 +1,7 @@
+# Globus Compute SDK
+
+[Globus Compute](https://globus-compute.readthedocs.io/en/latest/) is a distributed Function as a Service (FaaS) platform that enables flexible, scalable, and high performance remote function execution. Unlike centralized FaaS platforms, Globus Compute allows users to execute functions on heterogeneous remote computers, from laptops to campus clusters, clouds, and supercomputers.
+
+This package contains the Python SDK for interacting with Globus Compute. Notable functionality includes submitting functions to remote endpoints via the [executor](https://globus-compute.readthedocs.io/en/latest/executor.html), and querying endpoint status.
```

Comment:
I think there normally isn't a ',' when there is a 'single item and something' format?

Reply:
You can add commas to lists with two items if it adds clarity - see [here](https://www.grammar-monster.com/lessons/commas_in_lists.htm#:~:text=Commas%20with%20Two%20List%20Items,Please%20buy%20bread%20and%20milk.). I'm not in love with the phrasing here, though, so alternative suggestions are welcome.

Reply:
I thought it was slightly better without the ',' ie. ...executor.html) and querying endpoint... but it doesn't matter too much.